.travis.yml

```diff
@@ -11,7 +11,7 @@ matrix:
       env: SCRIPT="util/run-inside-docker.sh ubuntu-16.04 util/travis-ci/default-build.sh"
     # Default build on osx
     - os: osx
-      compiler: clang
+      language: generic
```

Comment:
why was this change required?

Reply:
This is needed to prevent Travis to set `CXX` and `CC` environmental variables. The automatic detection of the compiler works only if those two are not set.